UDPClientGUI/Events/ClientLoginEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace UDPCommGUI
+{
+    public class ClientLoginEvent
+    {
+        public ClientLoginEvent(string username, string password)
+        {
+            this.Username = username;
+            this.Password = password;
+        }
+
+        public readonly string Username;
+        public readonly string Password;
```

Comment:
Make these into properties without setters.

Programatically, it's not wrong but by and large in OO, don't want to expose fields as a convention.